src/components/User/Users/components/UserAccountDetails.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <div class="users-show__details-wrp">
+      <user-details
+        :id="id"
+        @reviewed="$emit(reviewed)"
```

Comment:
What is `reviewed`? This `$emit` goes to the emptiness. Does can it broke some of our functional?